client/src/components/toyProblems/StudentPrDetails.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Label, Card, Icon, Button, Header } from 'semantic-ui-react';
+
+export default class StudentPrDetails extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showAllAttemptedToyProblems: false
+    };
+    this.showHideDetails = this.showHideDetails.bind(this);
+  }
+
+  showHideDetails() {
+    const { showAllAttemptedToyProblems } = this.state;
+    this.setState({ showAllAttemptedToyProblems: !showAllAttemptedToyProblems });
+  }
+
+  render() {
+    const { pullRequestsList, selectedCohort } = this.props;
+    const { showAllAttemptedToyProblems } = this.state;
+
```

Comment:
Add error checking in case data isn't there.

For example, later you'll map through pullRequestList. You're checking if it's an array, which great! but, is it empty?

If empty, send a different component which alerts the user to the problem